tests/test_client_request.py

```diff
@@ -279,6 +279,34 @@ def test_host_header_ipv6_with_port(make_request: Any) -> None:
     assert req.headers["HOST"] == "[::2]:99"
 
 
+@pytest.mark.parametrize(
+    "url,headers,expected",
```

Comment:
It's visually nicer to have param names as separate string literals:
```suggestion
    ("url", "headers", "expected"),
```